docs/conf.py

```diff
@@ -0,0 +1,69 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+import datetime
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'slconf'
+copyright = f'20220-{datetime.date.today().year}, cameronwaterman< and contributors'
+author = 'cameronwaterman <>'
```

Comment:
I believe NI owns the copyright. We use this setting in the Python client repo:

```suggestion
copyright = f'2020-{datetime.date.today().year}, National Instruments'
author = 'National Instruments'
```

Reply:
Well that makes sense. Is it still National Instruments, or just NI now? Not sure since this is a legal-related.